lib/public/components/common/selection/infoLoggerButtonGroup/CopyToClipboardComponent.js

```diff
@@ -48,20 +48,19 @@ export class CopyToClipboardComponent extends StatefulComponent {
      * - `available`: True if the clipboard is available; false otherwise.
      * - `message`: A reason if the clipboard is not available, otherwise an empty string.
      */
-    isClipboardAvailable() {
+    checkClipboardAvailability() {
         if (!this.isContextSecure()) {
-            return { available: false, message: 'Clipboard not available in a non-secure context.' };
+            throw new Error('Clipboard not available in a non-secure context.');
         }
 
         if (!this.isClipboardSupported()) {
-            return { available: false, message: 'Clipboard API is not supported in this browser.',
-            };
+            throw new Error('Clipboard API is not supported in this browser.');
         }
 
         if (this.isWindowEmbedded()) {
-            return { available: false, message: 'Clipboard access is restricted in iframes.' };
+            throw new Error('Clipboard access is restricted in iframes.');
         }
-        return { available: true, message: '' };
+        return { available: true };
```

Comment:
You don't need to return anything in check, you simply run it and wait for it not to throw.